### ISSUE TITLE
File size issue (gcode > around 250Mb)

### ISSUE THREAD

**Issue by MlvPerry** [Snapmaker/Luban#726]
Hello, I have an issue with gcode files (for 3d printing) larger than approximately 250Mb.
They do not finish transfering to the Snapmaker 2.0 A350, or they crash the display software before starting the print.
I tried with different slicers and different files. For some prints is not possible to further reduce the file size via resolution decrease or print settings optimization...
I see other users have the same or very similar problems, for example [here](https://forum.snapmaker.com/t/file-transfer-issues/8618) and [here](https://forum.snapmaker.com/t/snapmaker-2-0-cura-profile/7537).

**Comment by scotthsl** [comment]
@ITmaze Yes, you are right, I am sorry for closing this issue directly before.
I asked my colleague to help to transfer it to luban here.

**Comment by MlvPerry** [comment]
Please note that the issue is not caused by the slicing software (or, to be precise, all of the slicing softwares I tried cause this same issue). So there is a problem with the Snapmaker software (firmware).
